packages/node/src/indexer/indexer.manager.ts

```diff
@@ -47,6 +48,7 @@ import { BlockContent } from './types';
 const { version: packageVersion } = require('../../package.json');
 
 const DEFAULT_DB_SCHEMA = 'public';
+const NULL_MERKEL_ROOT = hexToU8a('0x00');
```

Comment:
it is already defined somewhere right? can you group them together into a consts.ts